### PR TITLE
EWPP-2986: Drupal 10 compatibility.

### PR DESCRIPTION
**Pull request by brummbar**

### CODE REVIEW

**Review by drishu — DISMISSED** [review]

**Review by claudiu-cristea — APPROVED** [review]

**Review by drishu — APPROVED** [review]

File: .drone.yml [hunk 1]
```diff
@@ -25,29 +25,14 @@ services:
...
-    image: registry.fpfis.eu/fpfis/httpd-php:${PHP_VERSION}-ci
-    volumes:
-      - /cache:/cache
-    commands:
-      - composer self-update --2
-      - composer update --prefer-lowest --prefer-stable --ansi --no-progress
-    when:
-      matrix:
-        COMPOSER_BOUNDARY: lowest
+      - composer require drupal/core:^${CORE_VERSION} drupal/core-composer-scaffold:^${CORE_VERSION} --update-with-all-dependencies --ansi --no-progress --no-interaction
```
Comment (drishu) [inline .drone.yml:35]:
```suggestion
      - composer require drupal/core:~${CORE_VERSION} drupal/core-composer-scaffold:~${CORE_VERSION} --update-with-all-dependencies --ansi --no-progress --no-interaction
```

File: .drone.yml [hunk 2]
```diff
@@ -75,9 +60,10 @@ pipeline:
...
 matrix:
-  COMPOSER_BOUNDARY:
-    - lowest
-    - highest
-  PHP_VERSION:
-    - 8.0
-    - 8.1
\ No newline at end of file
+  include:
+    - CORE_VERSION: 9.4
```
Comment (drishu) [inline .drone.yml:64]:
```suggestion
    - CORE_VERSION: 9.4.0
```

File: .drone.yml [hunk 3]
```diff
@@ -75,9 +60,10 @@ pipeline:
...
+      PHP_VERSION: 8.0
+    - CORE_VERSION: 9.4
```
Comment (drishu) [inline .drone.yml:66]:
```suggestion
    - CORE_VERSION: 9.4.0
```

**Review by drishu — CHANGES_REQUESTED** [review]
noticed this only today in other PR, sry

**Review by drishu — APPROVED** [review]

**Review by sergepavle — APPROVED** [review]

**Pull request merged**